test/test_examples_1d_advection.jl

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Shouldn't we wrap this in a `module TestExamples1DAdvection`? IIRC, we used the modules to allow multiple inclusions of the test files.

Reply:
I used modules not because of possible problems when including the files multiple times but to not clutter the current namespace when including test files. Done in 741cf93.

Reply:
Thanks for the clarification.